modules/nw-sctp-about.adoc

```diff
@@ -0,0 +1,75 @@
+// Module included in the following assemblies:
+//
+// * networking/using-sctp.adoc
+
+[id="nw-sctp-about_{context}"]
+= Support for Stream Control Transmission Protocol (SCTP) on {product-title}
+
+As a cluster administrator, you can enable SCTP on the hosts in the cluster.
+On {op-system-first} the SCTP module is blacklisted by default.
+
+SCTP is a reliable message based protocol that runs on top of an IP network.
+
+When enabled, you can use SCTP as a protocol with both Pods and Services.
+A Service must be defined with the `type` parameter set to either the `ClusterIP` or `NodePort` value.
+
+[id="using-sctp-with-a-pod-or-service"]
```

Comment:
All of the anchor IDs in modules need to include `_{context}`. And I'd call this something like "Example Pod and Service configurations for SCTP" or something since gerunds should be used only with procedures.